src/hct_mis_api/apps/payment/services/payment_gateway.py

```diff
@@ -102,13 +102,17 @@ class PaymentSerializer(ReadOnlyModelSerializer):
     remote_id = serializers.CharField(source="id")
     record_code = serializers.CharField(source="unicef_id")
     payload = serializers.SerializerMethodField()
+    extra_data = serializers.SerializerMethodField()
 
-    def get_payload(self, obj: Payment) -> Dict:
+    def get_extra_data(self, obj: Payment) -> Dict:
         snapshot = getattr(obj, "household_snapshot", None)
         if not snapshot:
             raise PaymentGatewayAPI.PaymentGatewayAPIException(f"Not found snapshot for Payment {obj.unicef_id}")
 
-        snapshot_data = snapshot.snapshot_data
+        return snapshot.snapshot_data
```

Comment:
we want to send whole payment snapshot via API? each snapshot is like 1k line JSON